YAXLib/Deserialization.cs

```diff
@@ -808,6 +808,27 @@ private bool TrySetValueForEmptyElement(object obj, MemberWrapper member, Type m
         if (realType != null) memberType = realType;
     }
 
+    private IList? TryGetCollectionItemList(Type collectionItemType)
```

Comment:
Would be more clean to apply the "TryGet" pattern, that typically involves a method that attempts to retrieve a `out` value or object and returns a `boolean` indicating whether the retrieval was successful or not.